knative-operator/pkg/common/metrics.go

```diff
@@ -15,6 +15,7 @@ var (
 	)
 	KnativeServingUpG  = knativeUp.WithLabelValues("serving_status")
 	KnativeEventingUpG = knativeUp.WithLabelValues("eventing_status")
+	KnativeKafkaUpG    = knativeUp.WithLabelValues("kafka_status")
```

Comment:
cc @skonto another thing is added to Prometheus gauges

Reply:
Should we hold off on this until we actually enable to controller?

Reply:
whats the downside if we have that already here?

I assume this is likely merged after the 1.10 branch creation anyways :man_shrugging:

Reply:
Gonna do some changes and the whole CI will start from scratch. Let me comment these out too.

Reply:
Done